html/core/XCube_Utils.class.php

```diff
@@ -186,7 +186,7 @@ public static function decrypt($crypt_text, $key = null, $algorithm = 'des', $mo
             mcrypt_generic_deinit($td);
             mcrypt_module_close($td);
         } else {
-            $plain_text = openssl_decrypt($crypt_text, strtoupper($algorithm.'-'.$mode), $key, OPENSSL_ZERO_PADDING);
+            $plain_text = openssl_decrypt($crypt_text, strtoupper($algorithm.'-'.$mode), $key);
```

Comment:
Need OPENSSL_ZERO_PADDING to decrypting encrypt data by mcrypt (old data). Why remove it?